configuration/configurator/BuildSet.cpp

```diff
@@ -39,6 +40,9 @@ CBuildSetManager* CBuildSetManager::getInstance(const char* pBuildSetFile, const
         pBuildSetDirectory = s_pBuildSetManager->m_buildSetDir;
 
         s_pBuildSetManager->m_buildSetPath.setf("%s%s%s", pBuildSetDirectory, pBuildSetDirectory[strlen(pBuildSetDirectory)-1] == '/' ? "" : "/", pBuildSetFile);
+
+        if (! (CFile(s_pBuildSetManager->m_buildSetPath.str())).exists())
```

Comment:
it would be more normal and probably easier to read/follow the code if you just called checkFileExists() here